packages/react-reconciler/src/__tests__/ReactHooksWithNoopRenderer-test.js

```diff
@@ -3202,91 +3202,6 @@ describe('ReactHooksWithNoopRenderer', () => {
     });
   });
 
-  describe('useRef', () => {
```

Comment:
Why were these deleted?

Reply:
Can you update them to not read instead? Or to assert on the warning?

Reply:
All ref tests were moved into `useRef-test` since there are a lot of them. That big hooks test file kind of sucks.